lib/shared/log/tasks.js

```diff
@@ -1,7 +1,10 @@
 'use strict';
 
 var log = require('gulplog');
-var chalk = require('chalk');
+var stringWidth = require('string-width');
```

Comment:
Why do we need this module?

Reply:
This module is used to calculate the widths of task names or flag names with its configured styles. Those styles might include emojis, symbols, or other full width characters.